spring-core-5/src/main/java/com/baeldung/springbean/naming/component/Cat.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.springbean.naming.component;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+@Qualifier("cat")
+public class Cat implements Animal{
```

Comment:
minor formatting issue here with the opening parenthesis after implements

Reply:
fixex